### ISSUE TITLE
HFM core as a Service

### ISSUE THREAD

**Issue by harlam357** [harlam357/hfm-net#297]
- Thinking REST API... a lot of work to get there.

**Comment by harlam357** [comment]
Still on the path for this issue.  The first step is to get HFM working as a console app.  Fire, load configuration, and monitor.  Just without any of the WinForms GUI components.

**Comment by kbernhagen** [comment]
For the console app, I'd like to suggest https://github.com/migueldeicaza/gui.cs

**Comment by harlam357** [comment]
Yeah, I've seen that package @kbernhagen. My plan is to throw away the current WinForms (Windows only) UI and go full web based or PWA, probably Blazor Hybrid.
